### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by manuelstolze**
Bitte vorher überprüfen ob es bei euch mit der ORM-File klappt. Siehe DC.

### CODE REVIEW

File: GameRoom.ts [hunk 1]
```diff
@@ -26,15 +27,17 @@ export type createRoomOptions = {
...
 
-    createDate: Date;
+    createTime: Date;
 
     onCreate(options: any): void | Promise<any> {
         console.log(`[onCreate] Room created. Options: ${JSON.stringify(options)}`);
-        this.createDate = new Date();
+        this.createTime = new Date();
+
+        this.maxClients = 16;
```
Comment (Saphs) [inline GameRoom.ts:40]: why is there a maxClient value introduced?

File: entities/boardTile.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { type } from "node:os";
+import { Column, Entity, JoinColumn, ManyToMany, ManyToOne, OneToMany, PrimaryGeneratedColumn } from "typeorm";
+import Language from "./language";
+import User from "./user";
+
+@Entity()
+class BoardTile {
```
Comment (Saphs) [inline entities/boardTile.ts]: Didnt we have a boardTile class befor ? :o

File: package.json [hunk 3]
```diff
@@ -44,7 +46,7 @@
   "devDependencies": {
     "@types/cannon": "^0.1.5",
     "browserify": "^14.4.0",
-    "ts-node": "^8.10.2",
+    "ts-node": "^9.1.1",
```
Comment (Saphs) [inline package.json:53]: doesnt this imply breaking changes to existing code ? I am not sure if we should update this while changing large parts at the same time.

File: src/migration/1617312788972-InitMigration.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class InitMigration1617312788972 implements MigrationInterface {
```
Comment (Saphs) [inline src/migration/1617312788972-InitMigration.ts]: I dont like the name please replace all date numbers lol

File: src/entity/User.ts [hunk 5]
```diff
@@ -27,7 +27,7 @@ class User {
     public profile_picture: string;
 
     @Column({ type: "int" })
-    public figure: number;
+    public last_figure: number;
```
Comment (Saphs) [inline src/entity/User.ts]: next time clean your commits

Comment (manuelstolze) [reply src/entity/User.ts]: ok...

File: src/migration/1617312788972-InitMigration.ts (hunk 4 above)
Comment (manuelstolze) [reply src/migration/1617312788972-InitMigration.ts]: na.... LGTM!

File: package.json [hunk 6]
```diff
@@ -50,8 +50,8 @@
   "devDependencies": {
     "@types/cannon": "^0.1.5",
     "browserify": "^14.4.0",
-    "ts-node": "^9.1.1",
+    "ts-node": "^8.10.2",
```
Comment (Saphs) [inline package.json]: Please revert the originial section in the original commit and dont patch old commits with new ones. Its considered bad practise and leads to a confusing git history.

**Pull request merged**